deps/shims/command.node.ts

```diff
@@ -1,3 +1,4 @@
+// deno-lint-ignore-file
```

Comment:
I'd add `// deno-lint-ignore no-deprecated-deno-api` to the 2 `Deno.run` calls

Reply:
Ill get another warning if i do that B/c we still on older version. Maybe we can defer this change then?

`Ignore for code "no-deprecated-deno-api" was not used.deno-lint(ban-unused-ignore)`

Reply:
apologies, I did my review using `v1.33.x`